profiles/base/package.use.stable.mask

```diff
@@ -4,6 +4,10 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Zen <dev@pyl.onl> (2024-06-29)
+# All versions of UGRD on all architectures are currently unstable.
+sys-kernel/installkernel ugrd
+
```

Comment:
This file is no longer required due to the changes you made in 4238136b3b0ddbb9a62d6717902937cd3cd23f44

`arch/base/package.use.mask` overshadows `base/package.use.stable.mask` here